ejcIMC.py

```diff
@@ -25,6 +25,24 @@ def imcCalculate(statureValue, weigthValue):
     IMC = (weigthValue/(statureValue**2))
     return IMC
 
+def yourImc(imc):
+    if imc <= 16:
```

Comment:
16 : is a magic number.... try to do this variable declaration begining the code.... In variables section
and explain what do u refer to that.

In the future code maybe need to change that values.... it is better to change them at the beginning of the document than to be forced to read the whole project